meddlr/evaluation/recon_evaluation.py

```diff
@@ -97,7 +97,6 @@ def __init__(
         self._channel_names = channel_names
         self._structure_channel_by = structure_channel_by
         self._prefix = prefix
-
```

Comment:
nit: add this empty line back. If there is code that you changed and revert, it is generally good practice to revert to the exact same code that was there previously to avoid any extra changes in the changelist